README.md

```diff
@@ -298,6 +298,8 @@ Supported:
 
 Planned:
 
+* dnf
+* cargo
```

Comment:
Please add a brief mention in the commit message. Reason: the commit message wrt git history is insufficient unless you run `git show` on that particular commit.